runtime/doc/eval.txt

```diff
@@ -3665,7 +3668,8 @@ getftype({fname})					*getftype()*
 			getftype("/home")
 <		Note that a type such as "link" will only be returned on
 		systems that support it.  On some systems only "dir" and
-		"file" are returned.
+		"file" are returned.  On MS-Windows a symbolic link to a
+		directory returns "dir" instead of "link".
```

Comment:
I only checked Vim, not Neovim. But it shouldn't hold this up, win32 nvim is still alpha.